cdb2api/cdb2api.c

```diff
@@ -4067,12 +4040,15 @@ int cdb2_bind_param(cdb2_hndl_tp *hndl, const char *varname, int type,
 int cdb2_bind_index(cdb2_hndl_tp *hndl, int index, int type,
                     const void *varaddr, int length)
 {
-    int rc = 0;
     pthread_once(&init_once, do_init_once);
+    if (log_calls)
```

Comment:
We should log it at end and print rc

Reply:
ok, was trying to avoid the goto, but will do.
Thanks

Reply:
hah, there is no rc for this one, there might be for the other one i removed.

Reply:
I don't think there is any difference if we put at function entry.

Reply:
rc was removed from this function, we usually log rc when the function gives non-zero returns too.